resources/js/pages/bookshelf/add.vue

```diff
@@ -88,6 +89,11 @@ export default {
       } else {
         this.addBook(this.selectedBook)
       }
+    },
+    resetSelection(){
+      this.currentStep = 1
+      this.selectedBook = null
+      this.$refs.bookSelection.$data.searchText = ""
```

Comment:
Instead than set the SearchText, I would invoke the Method "resetSearch" available withint he BookshelfAddStep1. Also add some validation, just in case.

- [ ] Add JS validation to make sure that we just call the method if it is loaded (add an If(this.$refs.bookSelection && this.$refs.bookSelection.resetSearch)
- [ ] Change this line to call the "resetSearch" instead than reset just the string.

Reply:
I should have thought this earlier because just clearing the searchText will not toggle the disabled prop. Made these changes and amended them.